### ISSUE TITLE
Potential mappings in EFO

### ISSUE THREAD

**Issue by zoependlington** [EBISPOT/efo#451]
Maria Paula Magarinos has been looking at terms in EFO3 which may refer to the same disease, these should be reviewed. Note: Many of the cancer terms are due to incorrect synonyms or hierarchy.

-EFO:0009846 and HP:0031988 Spasm
-EFO:0000318 and MONDO:0024643 Myocardial Disease
-HP:0000863 and MONDO:0015790  Central Diabetes Insipidus
-EFO:0003841 and MONDO:0015074 Thyroid Gland Neoplasm
-HP:0002459 and HP:0012332  Dysautonomia
-MONDO:0000956 and EFO:0005588 Small Intestine Cancer
-MONDO:0001056 and EFO:0000178 Gastric Cancer
-MONDO:0001187 and EFO:0000292 Urinary Bladder Cancer
-MONDO:0001235 and EFO:0003880 Appendix Cancer
-MONDO:0001528 and EFO:0002921 Vulva Cancer
-MONDO:0001572 and EFO:0000731 Leiomyoma
-MONDO:0001657 and EFO:0003833 Brain Cancer
-MONDO:0001879 and EFO:0003835 Anus Cancer
-MONDO:0002013 and EFO:1001026 Lymphangioma
-MONDO:0002082 and EFO:0003769 Endocrine Gland Neoplasm
-MONDO:0002108 and EFO:0002892 Thyroid Cancer
-MONDO:0002129 and EFO:0003820 Bone Cancer
-MONDO:0002158 and EFO:1000251 Fallopian Tube Cancer
-MONDO:0002165 and EFO:1000657 Rectal Neoplasm
-MONDO:0002367 and EFO:0002890 Kidney Cancer
-MONDO:0002442 and HP:0001657 Long QT Syndrome
-MONDO:0002491 and EFO:0003890 Substance Abuse
-MONDO:0002512 and EFO:0000641 Papillary Adenocarcinoma
-MONDO:0002547 and EFO:0000760 Nerve Sheath Neoplasm
-MONDO:0002691 and EFO:0000182 Liver Cancer
-MONDO:0002715 and EFO:0003859 Uterine Cancer
-MONDO:0002974 and EFO:0001061 Cervical Cancer
-MONDO:0003046 and EFO:0003835 Anus Neoplasm
-MONDO:0003059 and EFO:0005540 Bile Duct Cancer
-MONDO:0003060 and EFO:0003891 Biliary Tract Cancer
-MONDO:0003235 and EFO:0009254 Optic Nerve Glioma
-MONDO:0004669 and EFO:0003826 Salivary Gland Cancer
-MONDO:0004857 and EFO:1001434 Tendinitis
-MONDO:0004938 and EFO:0003890 and Substance Dependence
-MONDO:0005232 and EFO:0003050 Large Cell Carcinoma
-MONDO:0005411 and Gallbladder Cancer
-MONDO:0005499 and EFO:0005543 Brain Glioma
-MONDO:0006058 and Orphanet:654 Wilms Tumor
-MONDO:0007576 and EFO:0002916 Esophageal Cancer
-MONDO:0007750 and EFO:0004911 Hypercholesterolemia, Familial
-MONDO:0008170 and EFO:0001075 Ovarian Cancer
-MONDO:0008315 and EFO:0001663 Prostate Cancer
-MONDO:0008627 and EFO:0003844 Ureter Cancer
-MONDO:0008903 and EFO:0001071 Lung Cancer
-MONDO:0009348 and EFO:0000183 Classic Hodgkin Lymphoma
-MONDO:0015075 and EFO:0002892 Thyroid Gland Carcinoma
-MONDO:0015492 and EFO:0004826 Anti-Neutrophil Cytoplasmic Antibody-Associated Vasculitis
-MONDO:0016587 and Orphanet:247 Arrhythmogenic Right Ventricular Cardiomyopathy
-MONDO:0016784 and Orphanet:59305 Gestational Trophoblastic Disease
-MONDO:0017858 and EFO:0000218 Acute Erythroid Leukemia
-MONDO:0019098 and EFO:0007160 Autoimmune Thrombocytopenia
-MONDO:0020669 and EFO:0003866 Paranasal Sinus Cancer
-MONDO:0021085 and EFO:0003897 Gastric Neoplasm
-MONDO:0021117 and EFO:0001071 Lung Neoplasm
-MONDO:0021118 and EFO:0007330 Intestinal Neoplasm
-MONDO:0021245 and EFO:0003868 Oral Cavity Neoplasm
-MONDO:0021258 and EFO:1000866 Choroid Neoplasm
-MONDO:0021632 and EFO:0003833 Primary Brain Neoplasm
-MONDO:0021698 and EFO:0003829 Alcohol-Related Disorders
-MONDO:0024621 and EFO:0003825 Serous Cystadenocarcinoma
-MONDO:0024645 and EFO:0007466 Retroperitoneal Neoplasm
-MONDO:0043243 and HP:0002745 Leukoplakia
-MONDO:0044334 and EFO:0000691 Connective and Soft Tissue Neoplasm
-MONDO:0044926 and EFO:1001931 Oropharyngeal Carcinoma
-MONDO:0044937 and EFO:1000657 Rectal Carcinoma

**Comment by paolaroncaglia** [comment]
@zoependlington 
FYI, the third case (HP:0000863 and MONDO:0015790 Central Diabetes Insipidus) is also in this list: https://github.com/monarch-initiative/mondo/issues/477
Looks like MONDO has identified it as an issue, but not resolved how to fix it yet.

**Comment by zoependlington** [comment]
I've started to look at these in a spreadsheet that can be found here: https://docs.google.com/spreadsheets/d/1qUx9W7RcARRQV87SBzVejhPkk87tnVn-DUEq0NjQsJ8/edit?usp=sharing

**Comment by zoependlington** [comment]
I reviewed synonyms and definitions of terms that seemed to be duplicates but were not.

I have added terms to the mapping files if they were a disease term mapping to a MONDO term.

I have added the MONDO term to the MONDO_duplicates file if they were phenotype terms mapping to a MONDO term.

Still to do:

- [ ] HP_0002459 (dysautonomia) and HP_0012332 (abnormal autonomic nervous system physiology)
     - dysautonomia term is obsolete in HP and replaced by the second term, mirror in EFO
- [ ] MONDO_0007750 (hypercholesterolemia, familial) duplicates in mondo with MONDO:0005439
     - inform MONDO
